upb/json/printer.c

```diff
@@ -208,18 +208,30 @@ static size_t fmt_bool(bool val, char* buf, size_t length) {
   return n;
 }
 
-static size_t fmt_int64(long val, char* buf, size_t length) {
+static size_t fmt_int32(long val, char* buf, size_t length) {
   size_t n = _upb_snprintf(buf, length, "%ld", val);
   CHKLENGTH(n > 0 && n < length);
   return n;
 }
 
-static size_t fmt_uint64(unsigned long long val, char* buf, size_t length) {
+static size_t fmt_uint32(unsigned long long val, char* buf, size_t length) {
```

Comment:
I think uint32 should use `unsigned int`.

Reply:
fmt_uint32 is also used for uint64 for map key